locale/af/toast.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Look example how the so called translation source file (af language) is filled in other components.

Key and actual value should be same for it to be in line with others.
E.g `"title": "",` --> `"title": "title",`

Reply:
This was fixed before.

Reply:
Value should not be in plural, so just `section.title` instead. _This is twice in the same file._

https://github.com/vrk-kpa/suomifi-design-system/blob/0bc89c22d852c7e3b32b69016680df7e6ab1981e/locale/af/toast.json#L8

Reply:
fixed